pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -297,9 +297,12 @@ def make_package(args):
     # Add extra environment variable file into tar-able directory:
     env_vars_tarpath = tempfile.mkdtemp(prefix="p4a-extra-env-")
     with open(os.path.join(env_vars_tarpath, "p4a_env_vars.txt"), "w") as f:
-        f.write("P4A_IS_WINDOWED=" + str(args.window) + "\n")
-        if hasattr(args, "orientation"):
-            f.write("P4A_ORIENTATION=" + str(args.orientation) + "\n")
+        try:
+            f.write("P4A_IS_WINDOWED=" + str(args.window) + "\n")
+            if hasattr(args, "orientation"):
+                f.write("P4A_ORIENTATION=" + str(args.orientation) + "\n")
+        except Exception as e:
```

Comment:
Don't we have a more precise exception to catch? It's always better to be sure the except clause catch the precise error that we are expecting, to avoid mismanaging other errors.

Reply:
oops sure, we need AttributeError here